src/svg/bookmars.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" width="32" height="32">
+  <g transform="translate(-11 -9)">
+    <path d="M34.9 34.5L26.8 31l-8.1 3.5V13.9h16.1v20.6zm0-23.8H18.8c-1.8 0-3.2 1.4-3.2 3.2v25.4l11.3-4.8 11.3 4.8V13.9c-.1-1.7-1.5-3.2-3.3-3.2z" fill-rule="evenodd" clip-rule="evenodd"/>
+  </g>
```

Comment:
A `<g>` seems useless here for just 1 `path`. This should work:

```suggestion
  <path d="M23.9 25.5L15.8 22l-8.1 3.5V4.9h16.1v20.6zm0-23.8H7.8C6 1.7 4.6 3.1 4.6 4.9v25.4l11.3-4.8 11.3 4.8V4.9c-.1-1.7-1.5-3.2-3.3-3.2z" fill-rule="evenodd" clip-rule="evenodd"/>
```

Reply:
i tried to remove this tags but is needed  to apply the transformation